src/sdk/object_sdk.js

```diff
@@ -286,6 +286,24 @@ class ObjectSDK {
 
     async _validate_bucket_namespace(data, params) {
         const time = Date.now();
+        // stat check (only in bucketspace FS)
+        const ns_allow_stat_bucket = Boolean(this._get_bucketspace().stat_bucket);
+        if (ns_allow_stat_bucket) {
+            let current_stat;
+            try {
+                current_stat = await this._get_bucketspace().stat_bucket(params.name);
+            } catch (err) {
+                // originally this part was added in case a bucket is created and then deleted
+                // and we would get ENOENT, but decide that on any error we will not use the stat
+                dbg.log4('_validate_bucket_namespace got an error', err, 'ignoring...');
+            }
+            if (current_stat) {
```

Comment:
instead of adding the code in different if block can you add it inside try block where `stat_bucket()` throws error,